src/components/Requirements/RequirementGroup.vue

```diff
@@ -146,6 +146,13 @@ export default defineComponent({
       this.req.reqs.forEach(req => {
         if (req.minCountFulfilled < req.minCountRequired) {
           ongoing.push(req);
+        } else if (
+          req.additionalRequirements == null ||
+          Object.values(req.additionalRequirements).every(
+            it => it.minCountFulfilled >= req.minCountRequired
+          )
+        ) {
+          ongoing.push(req);
```

Comment:
Confused on whether the req should be pushed to `completed` in this case. 

If we do want to push this req to `ongoing`, shouldn't we negate the condition because the req would be ongoing if even one of the additional requirements is not above the minCountRequired?
```
!(Object.values(req.additionalRequirements).every(
            it => it.minCountFulfilled >= req.minCountRequired
))
```

I have the same bug - completing the main requirement makes the entire requirement complete:

https://user-images.githubusercontent.com/54298311/117983789-4643f800-b305-11eb-93a9-4c2113634fbe.mov

Reply:
Should be fixed now!